common/src/main/java/com/LubieKakao1212/opencu/OpenCUConfigCommon.java

```diff
@@ -95,7 +95,7 @@ public static class Defaults {
         public static final OpenCUConfigCommon.DispenserDeviceConfig VANILLA_DISPENSER_DEVICE = new DispenserDeviceConfig(90, 5.0, 5.0, 1.0, 500, 1.0);
         public static final OpenCUConfigCommon.DispenserDeviceConfig GOLDEN_DISPENSER_DEVICE = new DispenserDeviceConfig(180, 10.0, 360.0, 1.0, 300, 1.0);
         public static final OpenCUConfigCommon.DispenserDeviceConfig DIAMOND_DISPENSER_DEVICE = new DispenserDeviceConfig(120, 2.5, 360.0, 1.5, 750, 1.5);
-        public static final OpenCUConfigCommon.DispenserDeviceConfig NETHERITE_DISPENSER_DEVICE = new DispenserDeviceConfig(15, 1.5, 10.0, 2.0, 1500, 3.0);
+        public static final OpenCUConfigCommon.DispenserDeviceConfig NETHERITE_DISPENSER_DEVICE = new DispenserDeviceConfig(50, 0, 0.0, 5.5, 1500, 5.0);
```

Comment:
Low rotation speed was intentional, it was supposed to be more of a stationary turret which you put in strategic places and aim ahead of time

Reply:
I mean netherite is heavy so it makes sense to turn slowly

Reply:
That's fair, as long as this is configuarable (I believe it is now) this can be ignored. Thoooo I think 15 is far too flow, 30 was a good balance but I put 50 to make it work better with Quantum. Again no worries if this can now be configured seperatly

Reply:
2 shots Iron Golem
10 shots on a Warden 

So 50 health on unarmored targets.